doc/feature/compartmentReport.md

```diff
@@ -0,0 +1,422 @@
+Additions to compartmentReport to support ZeroEQ streams
+============
+
+## Motivation
+
+The current CompartmentReport lacks an implementation based on ZeroEQ as the
+SpikeReport does. There is a network plugin implementated with codash, but it
+has been deprecated. Since there's a new specification of the SpikeReport API
+ready for implementation, it makes sense to review the CompartmentReport
+interface to make it consistent with the new SpikeReport and prepare it for the
+ZeroEQ restrictions at the same time.
+
+This specification proposes a new interface for the CompartmentReport with
+these goals in mind.
+
+## API
```

Comment:
I think we have to discuss it in person. For the next time it may be better to do as you suggest, but changing the whole spec now to be a diff is going to be too much work for what I can see it's worth (it's going to be too much formatting work for no new content for a document that won't be touched after acceptance). You have to convince me that it will really pay off during the implementation.